docs/docs/run/5_resources.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Resources
+sidebar_label: Resources
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+
+## Compute
+
+You can configure compute resources for your service like this:
+
+<Tabs groupId="package-manager">
+  <TabItem value="dashboard" label="dashboard" default>
+
+
+![compute slider](/img/run/resources_1.png)
+
+  </TabItem>
+  <TabItem value="toml" label="toml">
+
+```
+[resources.compute]
+cpu = 62
+memory = 128
+```
+
+  </TabItem>
+</Tabs>
+
+If you are configuring the resources via the configuration file instead of using the dashboard, keep in mind that cpu and memory allocation need to follow the following rules:
+
+* The value of the `memory` parameter must be exactly 2,048 times the value of the cpu parameter.
```

Comment:
do you mean `2.048`?

Reply:
It is a decimal separator so I guess it depends on whether you prefer dots or commas for decimals. It is usually a contentious point but feel free to propose a change if you think using a point is better (I know US English favors dots)